README.md

```diff
@@ -1,19 +1,111 @@
-# Why Building Api
+# Why Build An API?
 
 ## Objectives
 
   1. Understand the value of providing an API
   2. Understand the design considerations when creating APIs
 
-## Outline
-
-  * Building APIs boils down to two different purposes.
-    * Building for external usage (Foursquare API, Github API, etc)
-      * Heavily documented
-      * Versioning!!
-      * Auth, rate control
-      * We're not covering this much in this unit
-    * Building for internal use
-      * Discuss the problem of internal apis and auth. Can't store a key on the client, how to do auth?
-      * CORs as a way to stop other domains from using your internal API
-      * Not going to cover auth in a major way in this unit
+## Lesson
+
+We've seen how to *consume* various APIs, now let's talk about
+*providing* one.
+
+The choice to build and provide an API boils down to one of two options:
```

Comment:
I really like this framing device -- it helps to focus what can be a kind of unwieldy topic.